src/pages/plugins/index.js

```diff
@@ -95,21 +101,28 @@ function PluginFooter({ keywords = [] }) {
 }
 
 function PluginComponent({ plugin }) {
+  const deprecatedCard = plugin.deprecatedMessage ? true : false;
+  const deprecatedMessage = deprecatedCard && plugin.deprecatedMessage;
```

Comment:
Once you add `isDeprecated` to the class, you can just destructure both from the object, like:
```js
const { deprecatedMessage, isDeprecated } = plugin;
```

Reply:
Implemeted the destructure and it is super nice.